app/Models/Post.php

```diff
@@ -10,6 +10,8 @@ class Post extends Model
 {
     use Taggable, SoftDeletes;
 
+    protected $table = 'posts';
```

Comment:
oh I was just following [this](https://laravel-news.com/how-to-add-tagging-to-your-laravel-app) but you're right, it is not needed :)